addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -90,7 +90,6 @@
     @include print-variable('h5-font-size', $h5-font-size);
     @include print-variable('h6-font-size', $h6-font-size);
     @include print-variable('font-size-base', $font-size-base);
-    @include print-variable('small-font-size', $small-font-size);
```

Comment:
We don't want to show that on the `:root` element anymore?

Reply:
Automatically shown because it is part of the website-values map (see a few lines above where we print the website-values map). Here it was overriding the map value with the actual final value it has after computation (the bootstrap final value) (it was actually not really required already but there for consistency).

Now, it is not linked to the bootstrap value anymore: `o-website-value('small-font-size')` is the value for the o_small class, configurable in the editor; `$small-font-size` is the `em` value for the bootstrap small class/tag.